tests/data/dag.yml

```diff
@@ -0,0 +1,7 @@
+
+steps:
+  data://garden/wpp/2019/standard_projections:
+    - data://meadow/wpp/2019/standard_projections
```

Comment:
Unfortunately, we have to use real steps here to avoid `no recipe for executing step`. The other tests use the entire real `dag.yml` though, so this is at least a slight improvement :)